rule/rule.go

```diff
@@ -996,7 +996,7 @@ func (r *Rule) sync() {
 func ShouldKeep(e bzl.Expr) bool {
 	for _, c := range append(e.Comment().Before, e.Comment().Suffix...) {
 		text := strings.TrimSpace(strings.TrimPrefix(c.Token, "#"))
-		if text == "keep" {
+		if text == "keep" || strings.HasPrefix(text, "keep: ") {
```

Comment:
Please also document this new capability under https://github.com/bazelbuild/bazel-gazelle#keep-comments.

Reply:
Updated in 708dd8b